### DISCUSSION TITLE
How to invesitgate different Partition Coefficient models using MoBi?

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#1377] [How-Tos]
Dear OSP community,

How can I estimate the lipophilicity parameter using different partition coefficient models within the Parameter Identification step using MoBi?

Thank you in advance!
Kind regards,
Aya

**Comment by tobiasK2001** [comment]
Dear Aya,

yes indeed this option seems not to be available MoBi. And it does not seem to be possible if you export the MoBI simulation into PK-Sim and start the PI from there. I tried with a dummy project and observed only a bit of a strange behavior. The option calculation methods variation can be chosen in PK-Sim, and if one ticks the option all the same one can chose the different partition coefficents
![image](https://user-images.githubusercontent.com/25585248/201296877-cb0ab850-98bb-4f6c-b243-5018272afef5.png)
However PK-sim says that it will be on optimization (Instead of 5). If one tries to start the PI it gives error. At least it is like that in version 9. Probably this is fixed in a higher Version already

But coming back to your project Aya, I think you have to optimize the lipophilicity using different partition coefficient models first in PK-Sim and than go to MoBi.

Hope this might help you.

Best, Tobias

**Reply by AyaSaleh90** [reply]
Dear @tobiasK2001 thanks a lot for trying out that dummy project. I will give it a try here, as I'm using the latest version now.  At the beginning of my project I tested those P.C models using PK-sim, but since I modified the model using MoBi and lipophilicity was one of the most sensitive parameters with other ones, I wanted to re-estimate it again to be somehow sure of my choice of the P.C. model.  

Kind regards, Aya

**Comment by PavelBal** [comment]
Partition coefficients can be changed in MoBi in the Molecules-Building block:

![image](https://user-images.githubusercontent.com/25903040/201336097-08c517ca-630d-484b-b4d1-ec4b444ed611.png)

They will be applied when you create a new simulation.

**Reply by AyaSaleh90** [reply]
Dear @PavelBal thank you so much for your quick answer, but the problem I'm facing with this solution is that I will need to manually create a lot of simulations (for a parent & 2 metabolites, with different dosing and phenotypes) and pull them all in one P.I. step to re-estimate the logP value.  that's why I was asking if there is an efficient and quicker way to do it, as in PK-Sim?
Thanks in advance,
Aya

**Reply by PavelBal** [reply]
Unfortunately this is (currently) the only way to change calculation methods in MoBi.

**Reply by AyaSaleh90** [reply]
Alright, thanks a lot @PavelBal
